.github/workflows/test-linux-gpu.yml

```
@@ -51,8 +51,11 @@ jobs:
         conda install \
           --yes \
           -c "pytorch-${CHANNEL}" \
-          -c nvidia "pytorch-${CHANNEL}"::pytorch[build="*${VERSION}*"] \
+          -c nvidia pytorch \
```

Let's separate the channels from the packages we want to install

```suggestion
          -c nvidia \
          pytorch \
```

I'm also ok with having both channels and packages on one line together